test/sparse/sparse.jl

```diff
@@ -262,9 +262,15 @@ end
         a = UpperTriangular(speye(5) + triu(0.1*sprandn(5, 5, 0.2)))
         b = sprandn(5, 5, 0.2)
         @test (maximum(abs.(a\b - Array(a)\Array(b))) < 1000*eps())
+        # test error throwing for bwdTrisolve
+        @test_throws DimensionMismatch a\eye(6)
         a = LowerTriangular(speye(5) + tril(0.1*sprandn(5, 5, 0.2)))
         b = sprandn(5, 5, 0.2)
         @test (maximum(abs.(a\b - Array(a)\Array(b))) < 1000*eps())
+        # test error throwing for fwdTrisolve
+        @test_throws DimensionMismatch a\eye(6)
```

Comment:
Perhaps replace `eye(6)` with `Matrix(1.0I, 6, 6)`? Otherwise lgtm! :)

Reply:
Please belay that half-asleep review comment. The right-hand-side's contents should not matter here, only type and shape IIUC? And `eye(6)` was simply the shortest invocation that provided the right type and shape? If so, perhaps `fill(1., 6)` (or with a second shape argument if necessary)? :)